gp-ecommerce-fields/gpecf-deduct-deposit.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Gravity Perks // GP eCommerce Fields // Deduct Deposit from Order Summary
+ * https://gravitywiz.com/documentation/gravity-forms-ecommerce-fields/
+ *
+ * Plugin Name:   GP eCommerce Fields - Deduct Deposit from Order Summary
```

Comment:
Can you remove the extra space before the plugin name?

Reply:
Done